src/CoreFx.Private.TestUtilities/src/System/RetryHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System
+{
+    public static partial class RetryHelper
+    {
+        private static readonly Func<int, int> s_defaultBackoffFunc = i => i * 100;
```

Comment:
Perhaps `Math.Min(SomeConstant, i * 100)` so that we don't accidentally end up waiting several minutes?

Reply:
Done.